src/sections/formBlock/FormBlock.tsx

```diff
@@ -0,0 +1,46 @@
+import topForm from "../../assets/formBlock/topForm.png";
+import downForm from "../../assets/formBlock/downForm.png";
+
+import s from "./FormBlock.module.scss"
+
+
+const FormBlock = () => {
+
+    return (
+        <div className={s.wrapper}>
+            <div className={s.background}>
+                <h1 className={s.title}>CONTACT US</h1>
+                <div className={s.formWrapper}>
+                    <form className={s.form} action="">
+                        <span className={s.span}>I'm interested in...</span>
+                        <div className={s.userChoice}>
+                            <div className={s.userChoice__top}>
+                                <button className={s.userChoice__top_item}>UI/UX design</button>
+                                <button className={s.userChoice__top_item}>Frontend</button>
+                                <button className={s.userChoice__top_item}>Graphic design</button>
+                            </div>
+                            <div className={s.userChoice__bottom}>
+                                <button className={s.userChoice__bottom_item}>Devops</button>
+                                <button className={s.userChoice__bottom_item}>Architecture</button>
+                            </div>
+                        </div>
```

Comment:
1. You don't need to make it in two containers - make them in one container and use grid of flexboxes to autowrapping. 
2. When you click on one of the buttons, the page will reload. You need a `useState` to control the active button. 
3. The form block is not responsive.